devnotes/scom_init_plan.md

```diff
@@ -0,0 +1,24 @@
+# About this document
+This document describes plans of implementing SCOM init steps 8.9 and 8.10
+
+# soc
+Here files that are directly responsible for soc
+- Add ibm directory
+  - Add power9 directory
+    - Add Kconfig specific to POWER9 family
+    - Add bootblock, and romstage that are responsible for cpu configuration
+    - Memlayout is already defined in Motherboard section,\
```

Comment:
Maybe it shouldn't be?

Reply:
@IgorBagnucki  Leave a note about this comment and I will merge it

Reply:
I have found `memlayout.ld` for other platforms inside "soc/<vendor>/<cpu>" directory, and haven't found it anywhere else inside "motherboard".

Reply:
@IgorBagnucki Please leave a note that it should be moved from mainboard and we will merge it

Reply:
@macpijan I have added a note